gradle.properties

```diff
@@ -1 +1,9 @@
-org.gradle.jvmargs=-Xmx3G
\ No newline at end of file
+org.gradle.jvmargs=-Xmx3G
+
+mc_version=1.20.5
+felis_version=1.3.0-alpha
```

Comment:
get the latest felis and kittens version from here: https://repsy.io/mvn/0xjoemama/public/felis/